app/_locales/en/messages.json

```
@@ -399,6 +399,9 @@
   "allow": {
     "message": "Allow"
   },
+  "allowMetaMaskToDetectTokens": {
+    "message": "Allow MetaMask to detect and display your tokens with autodetection.You’ll be able to:"
```

```suggestion
    "message": "Allow MetaMask to detect and display your tokens with autodetection. You’ll be able to:"
```

Add space between sentences